processor/src/operations/u32_ops.rs

```diff
@@ -351,6 +368,30 @@ mod tests {
         assert!(process.execute_op(Operation::U32addc).is_ok());
     }
 
+    #[test]
+    fn op_u32add3() {
+        let a = (rand_value::<u64>() as u32) as u64;
+        let b = (rand_value::<u64>() as u32) as u64;
+        let c = (rand_value::<u64>() as u32) as u64;
+        let d = (rand_value::<u64>() as u32) as u64;
```

Comment:
You should be able to do these a bit simpler now. For example:
```Rust
let a = rand_value::<u32>() as u64;
```

Reply:
Thanks, I missed this. Please see [738d30](https://github.com/maticnetwork/miden/pull/221/commits/738d304f6f6dc161fb4c2ecc534049dd9290fed6).